lib/main.dart

```diff
@@ -1,36 +1,35 @@
 import 'package:flutter/material.dart';
-import 'package:movie_card/pages/home_page.dart';
-import 'pages/movie_details.dart';
-import 'shared/movie.dart';
+import 'package:movie_card/src/presentation/pages/home_page.dart';
+
+import 'src/config/route/app_routes.dart';
+import 'src/data/models/movie_model.dart';
+import 'src/config/theme/app_themes.dart';
+import 'src/core/util/strings.dart';
+import 'src/presentation/pages/movie_details.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(
+    const MyApp(),
+  );
 }
 
 class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+  const MyApp({
+    super.key,
+  });
 
-  static const routeName = '/';
+  static const String routeName = '/';
```

Comment:
This name is not clear enough, it's too generic... What does it represent? Should it be here?
Why not calling it `initialRoute` and have it inside `AppRoutes`?

Reply:
No, it should't, so i erase it 😄